source/extensions/filters/http/aws_lambda/config.cc

```diff
@@ -49,8 +64,21 @@ Router::RouteSpecificFilterConfigConstSharedPtr
 AwsLambdaFilterFactory::createRouteSpecificFilterConfigTyped(
     const envoy::extensions::filters::http::aws_lambda::v3::PerRouteConfig& proto_config,
     Server::Configuration::ServerFactoryContext&, ProtobufMessage::ValidationVisitor&) {
-  return std::make_shared<const FilterSettings>(FilterSettings{
-      proto_config.invoke_config().arn(), proto_config.invoke_config().payload_passthrough()});
+  InvocationMode invocation_mode;
+  switch (proto_config.invoke_config().invocation_mode()) {
```

Comment:
This is a copy of above, I would make a small helper for this.

Reply:
sure thing